src/content/docs/blog/tauri-1-7.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Announcing Tauri 1.7.0
+date: 2024-07-01
+authors: lucasfernog
+excerpt: The Tauri team is happy to announce the 1.7.0 release.
+banner:
+  content: |
+    You're reading the blog on the prerelease site for Tauri 2.0 -
+    <a href="https://tauri.app">Go to the Tauri 1.0 site</a>
+---
+
+import CommandTabs from '@components/CommandTabs.astro';
+
+The Tauri team is happy to announce the 1.7 release. This version includes several bug fixes, performance improvements and features backported from the upcoming v2 release.
+
+## Upgrading
+
+Make sure to update both NPM and Cargo dependencies to the 1.7.0 release. You can update the dependencies with:
+
+<CommandTabs
+  npm="npm install @tauri-apps/cli@latest @tauri-apps/api@latest"
+  yarn="yarn upgrade @tauri-apps/cli @tauri-apps/api --latest"
+  pnpm="pnpm update @tauri-apps/cli @tauri-apps/api --latest"
+  cargo="cargo update"
```

Comment:
Might as well tell people to also update the CLI
```suggestion
  cargo="cargo update && cargo install tauri-cli"
```

Reply:
this will confuse users who don't use the rust cli